apps/cookbook/src/app/examples/list-example/examples/items.ts

```diff
@@ -13,7 +13,7 @@ export const ListItemsExampleTemplate = `<kirby-list [items]="items" [showDivide
   // tslint:disable-next-line
   selector: 'cookbook-list-items-example',
   template: `
-    <kirby-page title="Items">
+    <kirby-page title="Items with no dividers">
```

Comment:
I would prefer if we can rename the example components so items is the one with dividers, and the one used in the no dividers list will be called NoDividersExampleComponent or something like it.

Reply:
I think the file name of this one should be changed too, to no-dividers. 
Sorry, just think we should be consistent around this so we are not confusing ourselves down the line.